config/routes.rb

```diff
@@ -6,28 +6,20 @@
   resources :tweets, except: %i[edit update] do
     member do
       post :retweet
-      get  :visitors, as: :visitor
+      get  :visitor, as: :visitor
       post :reply
       post :like
-      get :show_likes
+      get :likes
     end
   end
 
-  resources :profiles
-
-  # likes
-  # post 'like/:id', to: 'tweets#like', as: :like
-  # get 'like/:id', to: 'tweets#like_index'
-
-  # visitor counts
-  # get 'visitor/:id', to: 'tweets#visitors', as: :visitor
-
-  # follow unfollow
-  post 'profiles/follow', to: 'profiles#follow'
-  delete 'profiles/', to: 'profiles#unfollow'
-
-  # list of followers
-  get 'profiles/:id/follower', to: 'profiles#follower', as: :follower
+  resources :profiles do
+    member do
+      post :follow
+      get :follower
```

Comment:
@amnindersingh12 Have you rechecked the follow/unfollow feature after adding this route change?

Reply:
Yes, I did recheck this feature. There was a mistake in my unfollow feature. I have fixed it now, earlier it was 
 `<%= button_to "Unfollow", profiles_path(id: @profile.id), method: :delete`.